docs/admin/networking/troubleshooting.md

```diff
@@ -95,9 +95,8 @@ the NAT configuration, or deploy an internal STUN server.
 
 If a network interface on the side of either the client or agent has an MTU
 smaller than 1378, any direct connections form may have degraded quality or
-performance, as IP packets are fragmented. `coder ping` will indicate if this is
-the case by inspecting network interfaces on both the client and the workspace
-agent.
+might hang entirely. `coder ping` will indicate if this is the case by inspecting
+network interfaces on both the client and the workspace agent.
```

Comment:
this can go in as-is, but I think we can improve it with:

~~~suggestion
might hang entirely.

Use `coder ping` to check for MTU issues, as it inspects
network interfaces on both the client and the workspace agent:

```console
$ coder ping my-workspace
...
Possible client-side issues with direct connection:

 - Network interface utun0 has MTU 1280 (less than 1378), which may degrade the quality of direct connections or render them unusable.
```
~~~